Fails packaging when binary files

If you vendor charts so that it already contains tgz files the plugin will attempt to process tgz files for property substitution. It should only substitute extensions `(json|tpl|yaml|yml)`.

```
[INFO] Clear target directory to ensure clean target package
[INFO] Created target helm directory
[DEBUG] Processing helm files in directory /Users/steven/projects/hedera/hedera-mirror-node/charts/hedera-mirror
[DEBUG] Processing helm file /Users/steven/projects/hedera/hedera-mirror-node/charts/hedera-mirror/.DS_Store
[DEBUG] Copying to /Users/steven/projects/hedera/hedera-mirror-node/target/helm/Users/steven/projects/hedera/hedera-mirror-node/charts/hedera-mirror/.DS_Store
[DEBUG] Processing helm file /Users/steven/projects/hedera/hedera-mirror-node/charts/hedera-mirror/requirements.yaml
[DEBUG] Copying to /Users/steven/projects/hedera/hedera-mirror-node/target/helm/Users/steven/projects/hedera/hedera-mirror-node/charts/hedera-mirror/requirements.yaml
[DEBUG] Processing helm file /Users/steven/projects/hedera/hedera-mirror-node/charts/hedera-mirror/Chart.yaml
[DEBUG] Copying to /Users/steven/projects/hedera/hedera-mirror-node/target/helm/Users/steven/projects/hedera/hedera-mirror-node/charts/hedera-mirror/Chart.yaml
[DEBUG] Processing helm file /Users/steven/projects/hedera/hedera-mirror-node/charts/hedera-mirror/charts/hedera-mirror-grpc-0.1.0.tgz
[DEBUG] Copying to /Users/steven/projects/hedera/hedera-mirror-node/target/helm/Users/steven/projects/hedera/hedera-mirror-node/charts/hedera-mirror/charts/hedera-mirror-grpc-0.1.0.tgz
[DEBUG] Processing helm file /Users/steven/projects/hedera/hedera-mirror-node/charts/hedera-mirror/charts/hedera-mirror-importer-0.1.0.tgz
[DEBUG] Copying to /Users/steven/projects/hedera/hedera-mirror-node/target/helm/Users/steven/projects/hedera/hedera-mirror-node/charts/hedera-mirror/charts/hedera-mirror-importer-0.1.0.tgz
[DEBUG] Processing helm file /Users/steven/projects/hedera/hedera-mirror-node/charts/hedera-mirror/charts/prometheus-adapter-2.3.1.tgz
[DEBUG] Copying to /Users/steven/projects/hedera/hedera-mirror-node/target/helm/Users/steven/projects/hedera/hedera-mirror-node/charts/hedera-mirror/charts/prometheus-adapter-2.3.1.tgz
[DEBUG] Processing helm file /Users/steven/projects/hedera/hedera-mirror-node/charts/hedera-mirror/charts/prometheus-operator-8.13.2.tgz
[DEBUG] Copying to /Users/steven/projects/hedera/hedera-mirror-node/target/helm/Users/steven/projects/hedera/hedera-mirror-node/charts/hedera-mirror/charts/prometheus-operator-8.13.2.tgz
[DEBUG] Processing helm file /Users/steven/projects/hedera/hedera-mirror-node/charts/hedera-mirror/charts/loki-stack-0.36.1.tgz
[DEBUG] Copying to /Users/steven/projects/hedera/hedera-mirror-node/target/helm/Users/steven/projects/hedera/hedera-mirror-node/charts/hedera-mirror/charts/loki-stack-0.36.1.tgz
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary for Hedera Mirror Node 0.10.0-rc1:
[INFO]
[INFO] Hedera Mirror Node ................................. FAILURE [  0.278 s]
[INFO] Hedera Mirror Node Importer ........................ SKIPPED
[INFO] Hedera Mirror Node Coverage ........................ SKIPPED
[INFO] Hedera Mirror Data Generator ....................... SKIPPED
[INFO] Hedera Mirror Node Protobuf ........................ SKIPPED
[INFO] Hedera Mirror Node GRPC API ........................ SKIPPED
[INFO] Hedera Mirror Node REST API ........................ SKIPPED
[INFO] Hedera Mirror Node Test ............................ SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  1.170 s
[INFO] Finished at: 2020-05-01T09:17:54-05:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal com.deviceinsight.helm:helm-maven-plugin:2.5.0-SNAPSHOT:package (default-cli) on project hedera-mirror-node: Error creating helm chart: Could not resolve property: 'W��H�y���сR,��E���h�M�8&��@���vC2:�m��n����w%��ܷ�&�������' used in file: '/Users/steven/projects/hedera/hedera-mirror-node/target/helm/Users/steven/projects/hedera/hedera-mirror-node/charts/hedera-mirror/charts/loki-stack-0.36.1.tgz' -> [Help 1]
org.apache.maven.lifecycle.LifecycleExecutionException: Failed to execute goal com.deviceinsight.helm:helm-maven-plugin:2.5.0-SNAPSHOT:package (default-cli) on project hedera-mirror-node: Error creating helm chart: Could not resolve property: 'W��H�y���сR,��E���h�M�8&��@���vC2:�m��n����w%��ܷ�&�������' used in file: '/Users/steven/projects/hedera/hedera-mirror-node/target/helm/Users/steven/projects/hedera/hedera-mirror-node/charts/hedera-mirror/charts/loki-stack-0.36.1.tgz'
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:215)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:156)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:148)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:117)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:81)
    at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:56)
    at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:128)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:305)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:192)
    at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:105)
    at org.apache.maven.cli.MavenCli.execute (MavenCli.java:957)
    at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:289)
    at org.apache.maven.cli.MavenCli.main (MavenCli.java:193)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:62)
    at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:566)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:282)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:225)
    at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:406)
    at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:347)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:62)
    at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:566)
    at org.apache.maven.wrapper.BootstrapMainStarter.start (BootstrapMainStarter.java:39)
    at org.apache.maven.wrapper.WrapperExecutor.execute (WrapperExecutor.java:122)
    at org.apache.maven.wrapper.MavenWrapperMain.main (MavenWrapperMain.java:61)
Caused by: org.apache.maven.plugin.MojoExecutionException: Error creating helm chart: Could not resolve property: 'W��H�y���сR,��E���h�M�8&��@���vC2:�m��n����w%��ܷ�&�������' used in file: '/Users/steven/projects/hedera/hedera-mirror-node/target/helm/Users/steven/projects/hedera/hedera-mirror-node/charts/hedera-mirror/charts/loki-stack-0.36.1.tgz'
    at com.deviceinsight.helm.PackageMojo.execute (PackageMojo.kt:94)
    at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo (DefaultBuildPluginManager.java:137)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:210)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:156)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:148)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:117)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:81)
    at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:56)
    at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:128)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:305)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:192)
    at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:105)
    at org.apache.maven.cli.MavenCli.execute (MavenCli.java:957)
    at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:289)
    at org.apache.maven.cli.MavenCli.main (MavenCli.java:193)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:62)
    at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:566)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:282)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:225)
    at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:406)
    at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:347)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:62)
    at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:566)
    at org.apache.maven.wrapper.BootstrapMainStarter.start (BootstrapMainStarter.java:39)
    at org.apache.maven.wrapper.WrapperExecutor.execute (WrapperExecutor.java:122)
    at org.apache.maven.wrapper.MavenWrapperMain.main (MavenWrapperMain.java:61)
Caused by: java.lang.IllegalStateException: Could not resolve property: 'W��H�y���сR,��E���h�M�8&��@���vC2:�m��n����w%��ܷ�&�������' used in file: '/Users/steven/projects/hedera/hedera-mirror-node/target/helm/Users/steven/projects/hedera/hedera-mirror-node/charts/hedera-mirror/charts/loki-stack-0.36.1.tgz'
    at com.deviceinsight.helm.PackageMojo.findPropertyValue (PackageMojo.kt:155)
    at com.deviceinsight.helm.PackageMojo.access$findPropertyValue (PackageMojo.kt:29)
    at com.deviceinsight.helm.PackageMojo$processHelmConfigFiles$processedFiles$2$$special$$inlined$use$lambda$1$1.invoke (PackageMojo.kt:126)
    at com.deviceinsight.helm.PackageMojo$processHelmConfigFiles$processedFiles$2$$special$$inlined$use$lambda$1$1.invoke (PackageMojo.kt:29)
    at kotlin.text.Regex.replace (Regex.kt:159)
    at com.deviceinsight.helm.PackageMojo$processHelmConfigFiles$processedFiles$2$$special$$inlined$use$lambda$1.invoke (PackageMojo.kt:124)
    at com.deviceinsight.helm.PackageMojo$processHelmConfigFiles$processedFiles$2$$special$$inlined$use$lambda$1.invoke (PackageMojo.kt:29)
    at kotlin.sequences.TransformingSequence$iterator$1.next (Sequences.kt:172)
    at com.deviceinsight.helm.PackageMojo$processHelmConfigFiles$processedFiles$2.invoke (PackageMojo.kt:166)
    at com.deviceinsight.helm.PackageMojo$processHelmConfigFiles$processedFiles$2.invoke (PackageMojo.kt:29)
    at kotlin.sequences.SequencesKt___SequencesKt$onEach$1.invoke (_Sequences.kt:1352)
    at kotlin.sequences.TransformingSequence$iterator$1.next (Sequences.kt:172)
    at kotlin.sequences.SequencesKt___SequencesKt.toCollection (_Sequences.kt:722)
    at kotlin.sequences.SequencesKt___SequencesKt.toMutableList (_Sequences.kt:752)
    at kotlin.sequences.SequencesKt___SequencesKt.toList (_Sequences.kt:743)
    at com.deviceinsight.helm.PackageMojo.processHelmConfigFiles (PackageMojo.kt:139)
    at com.deviceinsight.helm.PackageMojo.execute (PackageMojo.kt:73)
    at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo (DefaultBuildPluginManager.java:137)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:210)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:156)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:148)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:117)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:81)
    at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:56)
    at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:128)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:305)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:192)
    at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:105)
    at org.apache.maven.cli.MavenCli.execute (MavenCli.java:957)
    at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:289)
    at org.apache.maven.cli.MavenCli.main (MavenCli.java:193)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:62)
    at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:566)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:282)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:225)
    at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:406)
    at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:347)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:62)
    at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:566)
    at org.apache.maven.wrapper.BootstrapMainStarter.start (BootstrapMainStarter.java:39)
    at org.apache.maven.wrapper.WrapperExecutor.execute (WrapperExecutor.java:122)
    at org.apache.maven.wrapper.MavenWrapperMain.main (MavenWrapperMain.java:61)
[ERROR]
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
```

--- a/src/main/kotlin/com/deviceinsight/helm/PackageMojo.kt
+++ b/src/main/kotlin/com/deviceinsight/helm/PackageMojo.kt
@@ -30,6 +30,7 @@ class PackageMojo : AbstractHelmMojo() {
 
 	companion object {
 		private val PLACEHOLDER_REGEX = Regex("""\$\{(.*)}""")
+		private val SUBSTITUTED_EXTENSIONS = setOf("json", "tpl", "yml", "yaml")
 	}
 
 	@Parameter(property = "chartRepoUrl", required = false)
@@ -116,6 +117,11 @@ class PackageMojo : AbstractHelmMojo() {
 			log.debug("Copying to ${targetFile.absolutePath}")
 			targetFile.apply {
 				parentFile.mkdirs()
+			}
+
+			if (!SUBSTITUTED_EXTENSIONS.contains(file.extension)) {
+				file.copyTo(targetFile, true)
+				return@onEach
 			}
 
 			targetFile.bufferedWriter().use { writer ->

